orderer/consensus/smartbft/chain.go

```diff
@@ -419,6 +427,72 @@ func (c *BFTChain) HandleMessage(sender uint64, m *smartbftprotos.Message, metad
 	c.consensus.HandleMessage(sender, m)
 }
 
+func (c *BFTChain) getSuspectsFromSignatures(signatures []*smartbftprotos.Signature) []int32 {
```

Comment:
Why can we not use the same code as in `assembleSuspectsList` in the `assembler.go` ?